cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -295,8 +299,13 @@ button.o-video-player_play-btn,
         display: block;
         // Pushes button out of masked window when not playing.
         margin-top: @standard-btn-height__px;
+
+        &:focus {
```

Comment:
I don't think we need this block as the close button will pick up the standard focus rectangle of buttons now that it is unmasked.